### PR TITLE
[IPCT1-633] - Lambda to notify new proposals

### PR DESCRIPTION
**Pull request by dev-jotape**
This PR fixes [IPCT1-633] at https://impactmarket.atlassian.net/browse/IPCT1-633

## New
New lambda that get the new proposals and notify on discord

### CODE REVIEW

File: services/proposals/handler.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      break
+    }
+  }
+  
+  const promises = newProposals.map((proposal) =>
+    axios.post(process.env.DISCORD_WEBHOOK!, {
+      embeds: [{
+        type: 'rich',
+        title: 'New Proposal',
+        description: 'New proposal opened: ' + parseInt(proposal.id),
```
Comment (obernardovieira) [inline services/proposals/handler.ts:40]: Ask @psoaresbj on how to export this to prismic and consume from there.

**Review by obernardovieira — CHANGES_REQUESTED** [review]
Looks good overall.
Just rename all instances of `PACTDelegateStorageV1` to `PACTDelegateABI` and export text from the notification.

File: services/proposals/handler.ts (hunk 1 above)
Comment (dev-jotape) [reply services/proposals/handler.ts]: done

**Review by obernardovieira — APPROVED** [review]
Update the README with the needed env variables, or a .env example and it's good :muscle:

**Pull request merged**